test/cpp_extensions/extension.cpp

```diff
@@ -0,0 +1,30 @@
+#include <torch/torch.h>
+
+using namespace at;
+
+Tensor sigmoid_add(Tensor x, Tensor y) {
+  return x.sigmoid() + y.sigmoid();
+}
+
+struct MMModule {
```

Comment:
Nice! NB it would be nicer to not call this `Module`, since it's not really `nn.Module` and can't be used as a replacement.